api/webpack.config.babel.js

```diff
@@ -18,11 +18,26 @@ module.exports = {
         filename: '[name].js',
     },
     module: {
-        loaders: [
+        rules: [
             {
                 test: /\.js$/,
-                loader: 'babel-loader',
                 exclude: path.resolve(__dirname, 'node_modules'),
+                use: [{
+                    loader: 'babel-loader',
+                    options: {
+                        presets: [
+                            ['env', {
```

Comment:
Can you configure babel in the `.babelrc` file ?
`babel-loader` automatically read this file

Reply:
@Kmaschta yes, thanks)